tests/browseruse/login_helpers.py

```
@@ -0,0 +1,139 @@
+                extracted_content="BROWSERUSE_TEST_PASSWORD environment variable not set"
+            )
+        
+        # Clear any existing content and input password
+        await page.keyboard.press('Control+a')
+        await page.keyboard.type(password)
+        return ActionResult(success=True, extracted_content="Password entered securely")
+    
+    @controller.action('wait_2_seconds')
+    async def wait_2_seconds(page: Page) -> ActionResult:
+        """Wait for 2 seconds to let the page stabilize."""
+        import asyncio
```

Pylint (3.3.7)

[convention] Import outside toplevel (asyncio)

(C0415)